admin/links/class-link-reindex-post-service.php

```diff
@@ -0,0 +1,35 @@
+<?php
+/**
+ * @package WPSEO\Admin\Links\Reindex
+ */
+
+/**
+ * Class WPSEO_Link_Reindex_Post_Service
+ */
+class WPSEO_Link_Reindex_Post_Service {
+
+	/**
+	 * Reindexes the posts by REST request.
```

Comment:
Based on the name, I would expect it to reindex all posts, not only the unprocessed ones.